docs/sources/dashboards/build-dashboards/annotate-visualizations/index.md

```diff
@@ -19,96 +19,112 @@ you can get event description and event tags. The text field can include links t
+- Directly in the panel, using the built-in annotations query (add link)
+- Adding annotation queries for any data source, using the dashboard settings (add link)
+- Using the HTTP API
 
-### Add annotation
+This page explains the first two options; for information about using the HTTP API, refer to [Annotations API]({{< relref "../../../developers/http_api/annotations/" >}}).
+
+## Add annotations in panels
+
+Grafana comes with a native annotation store and the ability to add annotation events directly from the graph panel using the built-in annotation query(link here) that exists on all dashboards.
+
+### Add an annotation
 
 1. In the dashboard click on the Time series panel. A context menu will appear.
 1. In the context menu click on **Add annotation**.
    ![Add annotation context menu](/static/img/docs/time-series-panel/time-series-annotations-context-menu.png)
 1. Add an annotation description and tags(optional).
-   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-annotation.png)
 1. Click save.
 
 Alternatively, to add an annotation, Ctrl/Cmd+Click on the Time series panel and the Add annotation popover will appear
 
-### Add region annotation
+### Add a region annotation
 
 1. In the dashboard Ctrl/Cmd+click and drag on the Time series panel.
    ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-region-annotation.gif)
 1. Add an annotation description and tags(optional).
 1. Click save.
 
-### Edit annotation
+### Edit an annotation
 
 1. In the dashboard hover over an annotation indicator on the Time series panel.
    <!--![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-edit-annotation.gif)-->
 1. Click on the pencil icon in the annotation tooltip.
 1. Modify the description and/or tags.
 1. Click save.
 
-### Delete annotation
+### Delete an annotation
 
 1. In the dashboard hover over an annotation indicator on the Time series panel.
    <!--![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-edit-annotation.gif)-->
 1. Click on the trash icon in the annotation tooltip.
 
-### Built-in query
+## Add annotations through dashboard settings
 
-After you added an annotation they will still be visible. This is due to the built in annotation query that exists on all dashboards. This annotation query will
-fetch all annotation events that originate from the current dashboard and show them on the panel where they were created. This includes alert state history annotations. You can
-stop annotations from being fetched and drawn by opening the **Annotations** settings (via Dashboard cogs menu) and modifying the query named `Annotations & Alerts (Built-in)`.
+In the dashboard settings you can create new queries to fetch annotations from the native annotation store using the built-in data annotation data source or add query other data sources.
```

Comment:
Should we say Annotation settings and then explain how to get there?

Reply:
I found a nice starting explanation inside Grafana, and there is no mention about **Native** annotations. Should we still name them like this? I would be great to be consistent between this doc and Grafana Docs.
![image](https://github.com/grafana/grafana/assets/20256983/24dade50-c06b-4d75-adfe-38ee1d8352ae)

Reply:
How to get to the annotation settings is explained in 73-75. And yeah, that explanation is actually really good. I'll add some of that into the introduction.